### DISCUSSION TITLE
/app/chromedriver as soon as opened tries to navigate to a crypto adv sites

### DISCUSSION THREAD

**Discussion by puppinoo** [FlareSolverr/FlareSolverr#1140] [Q&A]
Hi,
Just installed jackett + flaresolverr.
Noticed cpu spikes and found out this log. Is it normal?

flaresolverr     | 2024-04-02 17:17:45 DEBUG    ReqId 127388805703424 Started executable: `/app/chromedriver` in a child process with pid: 2198
flaresolverr     | 2024-04-02 17:17:45 DEBUG    ReqId 127388805703424 New instance of webdriver has been created to perform the request
flaresolverr     | 2024-04-02 17:17:45 DEBUG    ReqId 127388704818944 Navigating to... https://ad.a-ads.com/2313959?size=468x60
Who does that?

Thanks.

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/discussions/1138
